tc-vars.sh

```diff
@@ -164,9 +164,9 @@ NVCC_COMPUTE="3.5"
 
 ### Define build parameters/env variables that we will re-ues in sourcing scripts.
 if [ "${OS}" = "${TC_MSYS_VERSION}" ]; then
-    TF_CUDA_FLAGS="TF_CUDA_CLANG=0 TF_CUDA_VERSION=10.0 TF_CUDNN_VERSION=7 CUDA_TOOLKIT_PATH=\"${CUDA_INSTALL_DIRECTORY}\" CUDNN_INSTALL_PATH=\"${CUDA_INSTALL_DIRECTORY}\" TF_CUDA_COMPUTE_CAPABILITIES=\"${NVCC_COMPUTE}\""
+    TF_CUDA_FLAGS="TF_CUDA_CLANG=0 TF_CUDA_VERSION=10.0 TF_CUDNN_VERSION=7.5.0 CUDNN_INSTALL_PATH=\"${CUDA_INSTALL_DIRECTORY}\" TF_CUDA_PATHS=\"${CUDA_INSTALL_DIRECTORY}\" TF_ TF_CUDA_COMPUTE_CAPABILITIES=\"${NVCC_COMPUTE}\""
```

Comment:
What is this dangling `TF_` in the middle ?

Reply:
Typo, I'll remove it.